index.html

```diff
@@ -101,9 +108,61 @@ <h1 class="my-recent-works">My Recent Works</h1>
           <button class="work-card-button">See Project</button>
         </div>
       </div>
+
+      <!-- WORK 4 -->
+      <div class="work-item">
+        <div class="work-card">
+          <header class="work-card-title">
+            Multi-Post Stories<br>
+            Gain+Glory
+          </header>
+          <ul class="work-card-tags nobull">
+            <li class="work-card-tag">Ruby on rails</li>
+            <li class="work-card-tag">css</li>
+            <li class="work-card-tag">JavaScript</li>
+            <li class="work-card-tag">html</li>
+          </ul>
+          <button class="work-card-button">See Project</button>
+        </div>
+      </div>
+
+      <!-- WORK 5 -->
+      <div class="work-item">
+        <div class="work-card">
+          <header class="work-card-title">
+            Multi-Post Stories<br>
+            Gain+Glory
+          </header>
+          <ul class="work-card-tags nobull">
+            <li class="work-card-tag">Ruby on rails</li>
+            <li class="work-card-tag">css</li>
+            <li class="work-card-tag">JavaScript</li>
+            <li class="work-card-tag">html</li>
+          </ul>
+          <button class="work-card-button">See Project</button>
+        </div>
+      </div>
+
+      <!-- WORK 6 -->
+      <div class="work-item">
+        <div class="work-card">
+          <header class="work-card-title">
+            Multi-Post Stories<br>
+            Gain+Glory
+          </header>
+          <ul class="work-card-tags nobull">
+            <li class="work-card-tag">Ruby on rails</li>
+            <li class="work-card-tag">css</li>
+            <li class="work-card-tag">JavaScript</li>
+            <li class="work-card-tag">html</li>
+          </ul>
+          <button class="work-card-button">See Project</button>
+        </div>
+      </div>
     </section>
```

Comment:
- [ ] Kindly ensure that you stick as much as possible to the provided Figma design. Currently, the gap between the individual cards is big which makes the margins/ padding left and right of the cards not to match  the provided Figma design. Kindly look into it.

**This is how I am seeing this section:**

![image](https://user-images.githubusercontent.com/86318284/194056247-d4f0eb3d-2e30-4d30-ab6c-af93391c231d.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/194056361-da9b2fb2-4744-4a1a-afc9-85305431e8d5.png)

Reply:
Thank you! I've fixed gaps between grid items using width as 87%